requirements.txt

```diff
@@ -0,0 +1,9 @@
+# Core modules (mandatory)
+jupyterlite-core==0.4.4
```

Comment:
Maybe we can also add a kernel, for example `jupyterlite-pyodide-kernel==0.4.4`?